src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany
+    @JoinColumn(name = "order_id")
+    private List<Ticket> tickets;
+    private LocalDateTime dateTime;
+
+    @ManyToOne
+    private User user;
```

Comment:
Probably need to fetch lazy as well, and fetch them in orderDao method

Reply:
I made it on purpose as Order closely connected to User and it makes sense to load User with the order at once and User doesn't have any custom fields so will not load too much unnecessary data .